src/view.rs

```diff
@@ -0,0 +1,61 @@
+use std::process::Command as CliCommand;
+
+use conrod::backend::glium::glium;
+use conrod::backend::glium::glium::backend::glutin::glutin::Event;
+
+use crate::commands::Command;
+
+pub enum SpacerunEvent<'a> {
```

Comment:
Should I use a Box here to remove the lifetime annotation?

Reply:
that would mean copying the whole Command with its whole subtree, since the Box has to own its contents. Of course that probably doesn't make a lot of difference here performance-wise, but it's... inelegant ;) You could work with `Arc<Command>` everywhere instead, but I think the lifetime here is fine, since SpacerunEvent is a very temporary structure just used as a return value (it's not going to be stored anywhere)